keylime/da/examples/sqldb.py

```diff
@@ -17,14 +17,14 @@
 
 class AttestationRecord(TableBase):
     __tablename__ = "AttestationRecord"
-    time = sqlalchemy.Column(sqlalchemy.DateTime, primary_key=True)
+    time = sqlalchemy.Column(sqlalchemy.Integer, primary_key=True)
```

Comment:
there should be a migration?

Reply:
Good question. I have been treating these `examples` as suggestions for implementation of backends (I am even keeping these out of `pylint` and `pyright` for now, even though I run the former internally) and thus do not plan to have `alembic` migration for this particular backend. If, at some point, we decide to "promote" it from "example" to "reference implementation", yes I will write the proper migration scripts.

Reply:
Agreed, I don't think we need a migration yet